bin/_duo

```diff
@@ -66,6 +67,8 @@ var program = new Command('duo')
   .option('-C, --no-cache', 'enable or disable the cache during build', true)
   .option('-d, --development', 'include development dependencies.', false)
   .option('-g, --global <name>', 'expose entry as a global <name>', null)
+  .option('-i, --inline-source-map', 'include source-maps inline to the entry file', false)
```

Comment:
hmm okay, no big deal :)

I just thought two source map options may be a bit much :p